benchmark/flexsn/benchmark_vgg_inductor.py

```
@@ -110,30 +110,22 @@ def main() -> None:
          lambda: build_vgg(make_flexsn_factory())),
     ]
 
-    scale_label = "ImageNet-scale" if H >= 224 and W >= 224 else "CIFAR-scale"
-    print(f"Input : T={T}, B={B}, {C}x{H}x{W}  ({scale_label})")
-    print("Model : SpikingVGG-16-BN")
+    print(f"Input : T={T}, B={B}, {C}×{H}×{W}  (ImageNet-scale)")
```

Ruff (0.15.12)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)